imports/plugins/core/taxes/server/methods/methods.js

```diff
@@ -124,6 +125,36 @@ export const methods = {
     });
   },
 
+  /**
+   * @name "taxes/updateTaxCode"
+   * @method
+   * @memberof Methods/Taxes
+   * @summary updates the taxcode on options of a product.
+   * @param  {String} products array of products to be updated.
+   * @return {undefined} returns nothing
+   */
+  "taxes/updateTaxCode"(products) {
```

Comment:
Was there a reason to make this a Meteor method rather than just directly importing the method? If we are going to make it a Meteor method, shouldn't we add some sort of permission check?

Reply:
@zenweasel This method is being called from the client side code. Shouldn't methods that get called from the client side be "Meteor" methods?
I am adding the permission check.